restapi.go

```diff
@@ -508,22 +508,25 @@ func (s *Session) UserChannelPermissions(userID, channelID string) (apermissions
 // Calculates the permissions for a member.
 // https://support.discord.com/hc/en-us/articles/206141927-How-is-the-permission-hierarchy-structured-
 func memberPermissions(guild *Guild, channel *Channel, userID string, roles []string) (apermissions int) {
+
 	if userID == guild.OwnerID {
 		apermissions = PermissionAll
 		return
 	}
 
 	for _, role := range guild.Roles {
 		if role.ID == guild.ID {
-			apermissions |= role.Permissions
+			p, _ := strconv.Atoi(role.Permissions)
```

Comment:
See comment on `Role.Permissions` below. This would help clean up a lot of these unchecked errors, and means we would only pay the string to integer conversion cost once.